README.md

```diff
@@ -0,0 +1,191 @@
+
+### Changes
+
+#### Vertical whitespace
+
+There must be no vertical whitespace before the first member and after the last one of a class.
+
+```java
+public class ClassExample {
+  private int firstMember; // no vertical whitespace before this line
+
+  // ...
+
+  public void lastMember() {
+    // ...
+  } // no vertical whitespace after this line
+}
+```
+
+Multiple vertical whitespaces must never be used.
+
+#### Array initializers
+
+The preferred array initializer "block-like construct" style when it doesn't fit in one line is one value per line:
+
+```java
+new int[] {
+  0,
+  1,
+  2,
+  3,
+}
+```
+
+#### Numeric literals
+
+`float`-valued literals use a lowercase `f` suffix: `100.23f`.
+
+#### Class and interface names
+
+If it's a list of an entity, use singular: `CustomerListFragment` for instance.
+
+#### Field, method, variable and parameter names
+
+If you need to name a `List` of activities for example, just name it `activities` and if it's `ListView`: `activitiesListView`. So we use plural here.
+
+#### Local variable names
+
+One-character names are only permitted if they are `i`, `j` or `k` indexes in loops. They must be used in this respective order when nesting them. Example:
+
+```java
+for (int i = 0; i < n1; i++) {
+  for (int j = 0; j < n2; j++) {
+    for (int k = 0; k < n3; k++) {
+      // ...
+    }
+  }
+}
+```
+
+#### Type variable names
+
+A single capital letter must be used, such as `T`, `S` or `D`.
+
+#### Where Javadoc is used
+
+Javadoc must be taken into account only in projects that are an API, open source or if it's explicitly requested. It can also be used in places that looks necessary in order to clarify certain behavior.
+
+### Additions
+
+#### Conditional operator
+
+The conditional (ternary) operator may be used if the conditions and its corresponding values are considered to be simple enough. The readability of the code must not be disturbed. Also, it must not be used nested. Example:
+
+```java
+// This is okay:
+boolean minValue = a < b ? a : b;
+
+// This isn't okay:
+String userHasProperty = user != null && Objects.equals(user.getProperty(), "a sample value")
+    ? String.valueOf("%s %s %s", "Some", "complicated", "string")
+    : (user != null && user.satisfiesCondition() ? "this string" : "this other string");
+```
+
+#### Ask for the positive
+
+Following Martin Fowler's [Remove Double Negative](http://www.refactoring.com/catalog/removeDoubleNegative.html) refactoring, try to make the conditionals single positive.
+
+#### Interfaces naming
+
+Interfaces names must **not** have an `I` prefixed, like in `IComparable` (just `Comparable` is preferred). In the particular case of having an interface and an abstract class that implements it giving a default behavior, prefix the class with the work `Abstract`. Examples of this can be found in the Java API: `Queue` and `AbstractQueue`, `Set` and `AbstractSet`, among others.
```

Comment:
Question, shouldn't say "prefix the class with the word" instead of "prefix the class with the work"?